mix.exs

```diff
@@ -8,10 +8,11 @@ defmodule DataLogger.MixProject do
       app: :data_logger,
       version: @version,
       deps: deps(),
+      source_url: "https://github.com/Inflowmatix/data_logger",
       description:
-        "A toolkit for data mapping and language integrated query for ElixirA logger that can be used to log any kind of data to remote or local destinations",
```

Comment:
Typo here - `ElixirA` -> `Elixir`

Reply:
That's one of things that were fixed, actually.